releasenotes/notes/plot-hop-vs-ntrials-aa4b8d80808edfdb.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add :meth:`qiskit.ignis.verification.quantum_volume.QVFitter.plot_hop_accumulative` for plotting heavy output probability (HOP) vs number of trials similar to Figure 2a of QV64 paper (arXiv:2008.08571).
+    * HOP of individual trials are plotted as scatters.
+    * Cummulative HOP are plotted in red line.
+    * Two-sigma confidence intervals are plotted as shaded area.
+    * 2/3 success threshold is plotted as dashed line.
```

Comment:
Can you reformat this slightly. This will look quite weird in the rendered release notes:

![Screenshot_2020-09-22_14-54-41](https://user-images.githubusercontent.com/2447371/93925069-91ac8600-fce3-11ea-9e25-c16b1ac4c219.png) (which I just pulled from the artifact link on the ci results from the docs run: https://github.com/Qiskit/qiskit-ignis/pull/493/checks?check_run_id=1127699605)

Some quick suggestions, add a link for the arXiv paper and rework the bulleted list as sentences explaining how the factor into the visualization. If you want a bulleted list you'll need to add a newline before the first line (https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks ). Also can you wrap the lines at 100 characters, having a super long line like this will cause issues when this gets integrated into the combined documentation.

Reply:
I made the suggested changes. Please let me know if the release note is still not good.